dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcTestDataCreatorServiceTest.java

```diff
@@ -36,12 +36,14 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.HttpStatus;
 import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.test.context.junit.jupiter.EnabledIf;
 import uk.gov.justice.laa.crime.dces.integration.client.TestDataClient;
 import uk.gov.justice.laa.crime.dces.integration.maatapi.MaatApiClientFactory;
 import uk.gov.justice.laa.crime.dces.integration.maatapi.MaatApiWebClientFactory;
 import uk.gov.justice.laa.crime.dces.integration.maatapi.config.ServicesConfiguration;
 import uk.gov.justice.laa.crime.dces.integration.model.local.FdcAccelerationType;
 
+@EnabledIf(expression = "#{environment['sentry.environment'] == 'development'}", loadContext = true)
```

Comment:
This is a unit test of a utility used by the integration tests, right?
I guess it's tricky to know where to put it (since the tests/ and integrationTests/ source sets don't depend on each other at all), so this is probably the only possible solution.

Reply:
Yeah, I didn't want to mess with it's location for this PR, just wanted to add the same development requirement for it too. We can have do a pass for structure and have a single ticket to do so.